docker/php/docker-entrypoint.sh

```diff
@@ -6,60 +6,29 @@ if [ "${1#-}" != "$1" ]; then
 	set -- php-fpm "$@"
 fi
 
-if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
+if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'artisan' ]; then
 	# Install the project the first time PHP is started
 	# After the installation, the following block can be deleted
 	if [ ! -f composer.json ]; then
+	    echo "Download app at tmp/"
 		rm -Rf tmp/
-		composer create-project "symfony/skeleton $SYMFONY_VERSION" tmp --stability="$STABILITY" --prefer-dist --no-progress --no-interaction --no-install
+		composer create-project "laravel/laravel $LARAVEL_VERSION" tmp --stability="$STABILITY" --prefer-dist --no-progress --no-interaction --no-install --no-scripts
 
+        echo "Copy app in root directory"
 		cd tmp
-		composer require "php:>=$PHP_VERSION"
-		composer config --json extra.symfony.docker 'true'
 		cp -Rp . ..
 		cd -
 
+        echo "Remove tmp/"
 		rm -Rf tmp/
 
-		if grep -q ^DATABASE_URL= .env; then
-			echo "To finish the installation please press Ctrl+C to stop Docker Compose and run: docker compose up --build --wait"
-			sleep infinity
-		fi
-	fi
-
-	if [ "$APP_ENV" != 'prod' ]; then
+        echo "Run scripts"
+        composer run-script post-root-package-install
```

Comment:
Indentation issues?

Reply:
Very strange because in my IDE it's correctly indentig.

<img width="942" alt="Screenshot 2023-08-15 at 21 09 17" src="https://github.com/FunkyOz/laravel-docker/assets/26649880/2b5327fd-a2cd-402d-af40-f8ff06ff7456">